server/seed.py

```diff
@@ -21,7 +44,7 @@ def request(self, endpoint, method, token, data, out_field=None):
             resp = requests.request(
                 method=method,
                 headers=headers,
-                url=self.host + endpoint,
+                url=self.host + "/api/" + self.version + endpoint,
```

Comment:
This looks close to what I suggest in the previous comments. Something like this, but on the server code would be ideal.